src/v/redpanda/application.cc

```diff
@@ -144,6 +145,20 @@ static void set_local_kafka_client_config(
     }
 }
 
+static void set_pp_kafka_client_defaults(
+  pandaproxy::rest::configuration& proxy_config,
+  kafka::client::configuration& client_config) {
+    // override pandaparoxy_client.consumer_session_timeout_ms with
+    // pandaproxy.consumer_instance_timeout_ms
+    client_config.consumer_session_timeout.set_value(
+      proxy_config.consumer_instance_timeout.value());
+
+    if (!client_config.client_identifier.is_overriden()) {
+        client_config.client_identifier.set_value(
+          std::make_optional<ss::sstring>("pandaproxy_client)"));
```

Comment:
Is the closing parenthesis inside the string literal intentional?

Reply:
Good spot, fixed.